Rubberduck.Parsing/Symbols/ConcurrentlyConstructedDeclarationFinderFactory.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+using Rubberduck.Parsing.Annotations;
+using Rubberduck.VBEditor.Application;
+
+namespace Rubberduck.Parsing.Symbols
+{
+    public class ConcurrentlyConstructedDeclarationFinderFactory : IDeclarationFinderFactory
```

Comment:
Ninject is configured to inject a proxy type for abstract factories; only the interface is needed.